packages/e2e-contract-tests/playwright.config.ts

```diff
@@ -39,6 +39,7 @@ const config: PlaywrightTestConfig = defineConfig({
         channel: 'chrome-beta',
         ...devices['Desktop Chrome'],
       },
+      retries: IS_CI ? 4 : 0,
```

Comment:
CI shouldn't retry more than once to avoid long wait times. Please check if we still need this with the latest fixes